libs/examples/src/filter-field/BUILD.bazel

```diff
@@ -1,13 +1,32 @@
 load("//tools/bazel_rules:index.bzl", "ng_module")
 
-package(default_visibility = ["//:__subpackages__"])
+package(default_visibility = ["//visibility:public"])
 
 ng_module(
-    name = "filter-field",
-    srcs = glob(["**/*.ts"]),
-    angular_assets = glob(["**/*.html"]),
-    module_name = "@dynatrace/examples/filter-field",
-    tsconfig = "//libs/examples:tsconfig-lib",
+    name = "compile",
```

Comment:
this name is referenced in the e2e test app, which is why the build now fails on buildkite. 
We might need to change the reference in the e2e app 

https://github.com/dynatrace-oss/barista/blob/a59f8815b59c71b9e95f6a9e14d8b3ac4f762c49/apps/components-e2e/src/components/filter-field/BUILD.bazel#L18

Reply:
That's why the ci is failing :)